app/controllers/settings_controller.py

```diff
@@ -428,6 +427,8 @@ def _update_view_from_model(self) -> None:
             self.settings_dialog.sorting_alphabetical_radio.setChecked(True)
         elif self.settings.sorting_algorithm == "Topological":
             self.settings_dialog.sorting_topological_radio.setChecked(True)
+        if self.settings.dependency_for_sorting == "True":
```

Comment:
Just do
```suggestion
        if self.settings.dependency_for_sorting:
```
since its a bool in the settings model